lib/resty/http_connect.lua

```diff
@@ -160,22 +174,111 @@ local function connect(self, options)
         proxy_port = proxy_uri_t[3]
     end
 
+    local cert_hash
+    if ssl and ssl_client_cert and ssl_client_priv_key then
+        -- fallback to non-mTLS when any error
+        repeat
```

Comment:
Could we reduce the indent like this:
```
repeat
  if not ssl or 
     not ssl_client_cert or 
     not ssl_client_priv_key
  then
    break
  end
  -- fallback to non-mTLS when any error
  ...
until true
```

Reply:
The indent of this library is 4 spaces, so I chose to keep consistent with it.

Reply:
Sorry, My mean is that we can remove the big `if ssl ... then ... end` statement, only keep `repeat ... until true`.

Reply:
Uh-oh, got it.